Server/mods/deathmatch/logic/packets/CEntityAddPacket.cpp

```diff
@@ -1029,6 +1029,10 @@ bool CEntityAddPacket::Write(NetBitStreamInterface& BitStream) const
                                 vertex.data.vecPosition = *iter;
                                 BitStream.Write(&vertex);
                             }
+                            float fFloor, fCeil;
+                            pPolygon->GetHeight(fFloor, fCeil);
+                            BitStream.Write(fFloor);

```

Comment:
support needed for backwards compatibility. please refer to [https://github.com/multitheftauto/mtasa-blue/pull/240#issuecomment-406815791] for a description on how to do this